meta-sokol-flex-distro/lib/bblayers/flex_utils.py

```diff
@@ -295,7 +295,8 @@ def do_dump_licenses(self, args):
 
         # if append flag is set then read the existing file first and create a consolidated set
         if args.append:
-            with open(filename, 'r') as f:
+            with open(filename, 'a+') as f:
+                f.seek(0)
```

Comment:
Hmmm checking the existence of the file would make more sense in this case, no?
```
if args.append and os.path.exists(filename):
```
thoughts?

Reply:
Yes, that would be more appropriate. Thanks. Let me update the PR and do a quick validation.